docs/source/app-dev/bindings-ts/daml2js.rst

```diff
@@ -4,7 +4,7 @@
 Use the JavaScript Code Generator
 #################################
 
-The command ``daml codegen js`` generates JavaScript (and TypeScript) that can be used in conjunction with the `JavaScript Client Libraries <../index>`_ for interacting with a Daml ledger via the `HTTP JSON API <../json-api/index.html>`_.
+The command ``daml codegen js`` generates JavaScript (and TypeScript) that can be used in conjunction with the `JavaScript Client Libraries <https://docs.daml.com/app-dev/bindings-ts/index.html>`_ for interacting with a Daml ledger via the `HTTP JSON API <https://docs.daml.com/json-api/index.html>`_.
```

Comment:
We should not have absolute links to ourselves, that robs Sphinx of its ability to check links and make them relative in versioned sites.

Reply:
ok, i'll fix this and the others

Reply:
fixed